specs/Picker-API_spec.md

```diff
@@ -0,0 +1,115 @@
+# Background
+
+The current FilePicker API for UWP has several limitations. Some of the top limitations 
```

Comment:
One current limitation that customers have complained about is that you can't specify an arbitrary start location, just the enum values.